azure-devops/projects/io-pay-projects/io-functions-pay-portal.tf

```
@@ -0,0 +1,161 @@
+variable "io-functions-pay-portal" {
+  default = {
+    repository = {
+      organization   = "pagopa"
+      name           = "io-functions-pay-portal"
+      branch_name    = "main"
+      pipelines_path = ".devops"
+    }
+    pipeline = {
+      production_resource_group_name = "io-p-rg-internal"
+      production_app_name            = "io-p-func-iopayportal"
+      cache_version_id               = "v3"
+    }
+  }
+}
+
+# code review
+resource "azuredevops_build_definition" "io-functions-pay-portal-code-review" {
+  depends_on = [azuredevops_serviceendpoint_github.io-azure-devops-github-pr, azuredevops_project.project]
+
+  project_id = azuredevops_project.project.id
+  name       = "${var.io-functions-pay-portal.repository.name}.code-review"
+  path       = "\\${var.io-functions-pay-portal.repository.name}/io-functions-pay-portal"
+
+  pull_request_trigger {
+    initial_branch = var.io-functions-pay-portal.repository.branch_name
+    forks {
+      enabled       = false
+      share_secrets = false
+    }
+    override {
+      auto_cancel = false
+      branch_filter {
+        include = [var.io-functions-pay-portal.repository.branch_name]
+      }
+      path_filter {
+        exclude = []
+        include = []
+      }
+    }
+  }
+
+  repository {
+    repo_type             = "GitHub"
+    repo_id               = "${var.io-functions-pay-portal.repository.organization}/${var.io-functions-pay-portal.repository.name}"
+    branch_name           = var.io-functions-pay-portal.repository.branch_name
+    yml_path              = "${var.io-functions-pay-portal.repository.pipelines_path}/io-functions-pay-portal/code-review-pipelines.yml"
```

```suggestion
    yml_path              = "${var.io-functions-pay-portal.repository.pipelines_path}/io-functions-pay-portal/.devops/code-review-pipelines.yml"
```
https://github.com/pagopa/io-pay-portal/tree/main/io-functions-pay-portal/.devops
you need to rename code review file